<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WifiConnectTestClass</class>
 <widget class="QWidget" name="WifiConnectTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WifiConnectTest</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="scan_adapter">
       <property name="text">
        <string>刷新适配器列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>适配器：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="interfaceDescription">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scan_wifi">
       <property name="text">
        <string>扫描Wifi列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>适配器状态：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="interfaceStatus">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="disconnect_tag">
       <property name="text">
        <string>断开连接</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connect_tag">
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="WifiConnectTest.qrc"/>
 </resources>
 <connections/>
</ui>
